app/services/taxonomy/taxonomy_root_retrieval.rb

```diff
@@ -0,0 +1,8 @@
+module Taxonomy
+  class TaxonomyRootRetrieval
```

Comment:
I'm not sure what I think about the name here, while I guess this retrieves the homepage, which is the "Root" of the taxonomy, what it actually returns is the level one taxons. So maybe "LevelOneTaxons" would be a more fitting service name?

Reply:
Good point. I'll rename this to LevelOneTaxonsRetrieval - I think 'root' and 'levelone' is a bit inconsistent throughout the app now